gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterUtils.java

```diff
@@ -112,6 +116,28 @@ public static Path getJobStateFilePath(boolean usingStateStore, Path appWorkPath
     return jobStateFilePath;
   }
 
+  /**
+   * Set the system properties from the input {@link Config} instance
+   * @param config
+   */
+  public static void setSystemProperties(Config config) {
+    Properties properties = ConfigUtils.configToProperties(ConfigUtils.getConfig(config, GobblinClusterConfigurationKeys.GOBBLIN_CLUSTER_SYSTEM_PROPERTY_PREFIX,
+        ConfigFactory.empty()));
+
+    for (Map.Entry<Object, Object> entry: properties.entrySet()) {
+      if (entry.getKey().toString().equals("java.io.tmpdir")) {
```

Comment:
Shall we add `java.io.tmpdir` to a string constant ?

Reply:
Addressed.